packages/core/src/main/Account.ts

```diff
@@ -123,21 +123,16 @@ export class Account extends EventEmitter {
 
   /**
    * @param apiClient The apiClient instance to use in the core (will create a new new one if undefined)
-   * @param storeEngineProvider Used to store info in the database (will create a inMemory engine if undefined)
+   * @param storeEngineProvider Used to store info in the database (will create a inMemory engine if returns undefined)
    */
-  constructor(apiClient: APIClient = new APIClient(), storeEngineProvider?: StoreEngineProvider) {
+  constructor(
+    apiClient: APIClient = new APIClient(),
+    {createStore = () => undefined, enableMls}: {createStore?: CreateStoreFn; enableMls?: boolean} = {},
```

Comment:
I think i prefer capitalizing all of an abbreviation for readability `enableMLS` but this is just a suggestion

Reply:
Rigth, makes sense. Actually I should remove it altogether from this PR, since it's not the direct topic here. Good catch :)